platform/ios/resources/ja.lproj/Localizable.strings

```diff
@@ -0,0 +1,75 @@
+﻿/* Accessibility hint */
+"ANNOTATION_A11Y_HINT" = "詳細を伝える";
+
+/* No comment provided by engineer. */
+"API_CLIENT_400_DESC" = "The session data task failed. Original request was: %@";
+
+/* No comment provided by engineer. */
+"API_CLIENT_400_REASON" = "The status code was %ld";
+
+/* No comment provided by engineer. */
+"CANCEL" = "キャンセル";
+
+/* Accessibility hint */
+"COMPASS_A11Y_HINT" = "地図を真北に回転させる";
+
+/* Accessibility label */
+"COMPASS_A11Y_LABEL" = "磁針";
+
+/* Compass abbreviation for north */
+"COMPASS_NORTH" = "北";
+
+/* Instructions in Interface Builder designable; {key}, {plist file name} */
+"DESIGNABLE" = "To display a Mapbox-hosted map here, set %1$@ to your access token in %2$@\n\nFor detailed instructions, see:";
+
+/* Setup documentation URL display string; keep as short as possible */
+"FIRST_STEPS_URL" = "mapbox.com/help/first-steps-ios-sdk";
+
+/* Accessibility hint */
+"INFO_A11Y_HINT" = "Shows credits, a feedback form, and more";
+
+/* Accessibility label */
+"INFO_A11Y_LABEL" = "この地図について";
+
+/* Accessibility label */
+"LOGO_A11Y_LABEL" = "Mapbox";
+
+/* Accessibility label */
+"MAP_A11Y_LABEL" = "地図";
+
+/* Map accessibility value */
+"MAP_A11Y_VALUE" = "ズーム %1$dx\n%2$ld ピン現れる";
```

Comment:
Are you sure you want “x” to be read aloud after the zoom level number? In English, it would sound like “ten ex”. Not sure about Japanese.